content/changelog/2024/long-chats-improvements.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Cody v1.38: Improved Chat Performance"
+authors:
+  - name: Alex Isken
+    url: https://x.com/AlexanderIsken
+    avatar: https://storage.googleapis.com/sourcegraph-assets/blog/alex_avatar.png
+publishDate: 2024-10-14T10:00-02:00
+description: "Cody v1.38 brings significant improvements to chat performance and updates to the autocomplete provider. This release focuses on enhancing the user experience in long chats and streamlining autocomplete functionality."
+tags: [Cody, VS Code]
+version: [v1.38]
+versionIcon: /assets/changelog/vscode.svg
+slug: 'long-chats-improvements'
+published: true
+---
+
+We're excited to announce the release of Cody v1.38, which brings notable improvements to chat performance.
+
+## Improved Chat Performance
+
+In previous versions of Cody, some users experienced slowness with the Chat interface, particularly when conversations became lengthy or when Cody was responding with long replies. We've heard your feedback, and we're pleased to announce that these issues have been addressed in this release.
+
+We've made significant enhancements to the webview performance in long chats. These improvements are designed to provide a smoother and more responsive experience, especially when dealing with extended conversations. The changes implemented in [pull/5866](https://github.com/sourcegraph/cody/pull/5866), [pull/5875](https://github.com/sourcegraph/cody/pull/5875), and [pull/5879](https://github.com/sourcegraph/cody/pull/5879) collectively contribute to a more efficient chat interface, allowing users to navigate and interact with long chat histories more easily.
```

Comment:
I would suggest not having the pull request links unless you think they add value for a user to go in and manually read them. For the average user, I don't think they'll want to open them and interpret themselves